torchtext/csrc/register_bindings.cpp

```diff
@@ -0,0 +1,80 @@
+#include <regex_tokenizer.h>
+#include <torch/script.h>
+#include <vectors.h>
+#include <vocab.h>
+
+namespace torchtext {
```

Comment:
Are we moving sentencepiece here?

Reply:
Let me do that in the followup PR since I need to create a header file for sentencepiece